new-dti-website/components/team/MemberGroup.tsx

```diff
@@ -101,71 +87,14 @@ type MemberDetailsProps = {
 
 export const MemberDetails: React.FC<MemberDetailsProps> = (props: MemberDetailsProps) => {
   const [hover, setHover] = useState<boolean>(false);
+  const mouseHandler = () => setHover(!hover);
```

Comment:
```suggestion
  const mouseHandler = () => setHover(prev => !prev)
```
https://legacy.reactjs.org/docs/hooks-reference.html#functional-updates
https://ogzhanolguncu.com/blog/how-does-prevstate-works-under-the-hood